### DISCUSSION TITLE
A collection of balancing issues from the new medic talents

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#10397] [Unstable feedback]
The new unstable talents are quite good on the most part, though there are some in the medic tree that are unbalanced, underwhelming, or completely useless. 

**Tier 1**
This tier is dominated by two very clear best talents. The rest are either just useless or too underwhelming to be good picks. This was a much, much worse balancing issue before it was confirmed that all of the generic talents could be picked, but some of these talents are still a waste of a talent point. 

- Health Insurance (15% more xp and money when completing an escort mission)
This talent is quite weak. Escort missions are already relatively low paying, so 15% is a very small bonus. 15% bonus equates to an extra 165-375 marks for each escort mission, which aren't exactly super common. 
This talent can be improved by making it stronger. As it stands it's not really worth taking.

- La resistance (+35% opiate addiction resistance, +10% husk infection resistance)
Perhaps the most well-known "bad talent" in the unstable version, La Resistance is really awful. The +10% husk resistance added later was okay, but it's still terrible. It's terrible because opiate addiction is just never an issue, much less for the medic. It is non-lethal in 99% of cases, and curable with a single dose of naloxone, or just time. Even 100% resistance would still make it worse than most other talents in this tier. 
The Husk Infection resistance is also really underwhelming. Husk infection resistance does not decrease the chance of catching husk (even if most people assume that it does), so all this talent does is decrease the rate of husk infection by _0.03/sec_.
This talent desperately needs to be better.
Some afflictions that would be useful to have increased resistances for: Paralysis, Concussion, Nausea, Chem addiction (chem addiction because it's more common and cannot be cured)

- Nobody Important Dies (7.5% more mission xp if nobody dies)
This talent is commonly compared to the similar security talent Bootcamp, which is nearly _5 times better_. The 7.5% more xp from this talent is abysmally low, and really should be buffed to be closer to other xp boost talents.

- Stayin' Alive (Crewmates' CPR is 20% more effective at reducing oxygenlow)
For a game that is set underwater, suffocating to death in Baro is surprisingly difficult. Not only is oxygen incredibly abundant (a single stack of O2 lasts a bit over 5 minutes), but it takes over a minute and a half to die after falling unconscious, and for any crew member with a half-decent medical skill, CPR takes only a few seconds to instantly revive.
It is for this reason that any talent that increases CPR is really non-impactful. In most cases the medic will have ample time to get to reach a downed crewmate before they die.
This talent would either need to be much better, or reworked entirely, just because suffocating is so rare.

It's possible that these talents seem so weak just because the other two talents not mentioned (Example of Health and Self Care) are comparatively much better options.

**Tier 2**
This tier has very good balancing. There are only two things that to comment on:

- Dr. Submarine (Medical items you use are 25% more potent)
This is a really strong talent. Having this means using 20% less medicine, which is great value long term, and exceedingly useful in saving lives on short supply of resources. It's probably strong enough and thematically appropriate to be in the Medic tree instead, if needed.

- No Pressure (Unlock recipe: Pressure Stabilizers)
Pressure stabilizers are fun, however, are prohibitively expensive for their functionality. They provide up to 100 seconds of pressure immunity for the low price of 1.5 oxygenite shards. That's 0.9 shards per minute to do what a diving suit can do indefinitely. The talent is fine, but the item that it unlocks is simply not worth it.
Pressure stabilizers could be improved by decreasing their cost to fabricate (especially since oxygenite is harder to come across in this version), increasing the duration, and/or giving an extra buff on top. It would be appropriate to have a swim speed buff attached to the pressure stabilized affliction, (like oxygenite tanks).

**Xenologist**
This is a really solid tree. It's also perhaps the only tree in unstable which has its last talent not actually be its best one. Only a few worth talking about:

- Bloody business (alien blood gives 50% less psychosis, find more alien blood on large creatures)
This talent is extremely underwhelming. Alien blood is already extremely plentiful; every crawler, spineling, and many other common creatures drop it. A single extra one from a handful of creatures is not that impactful. Additionally, psychosis is fairly easy to deal with. The main problem with this talent is the alternative, Gene Splicer, is just leagues better, and will always be picked over Bloody Business. Could possibly be moved down to be in the same tier as Gene Harvester to avoid this issue, but is still overall a really weak talent. It could be improved by reducing the Psychosis even further, and/or increase the extra alien blood drops to affect all creatures.

- Gene Splicer (Refining genetic materials adds an additional 10% to their potency.)
Simply the best talent in this tree, (final talent included). The description of this talent does not do it justice, because the real effect of this talent is that it triples the average improvement when refining genes. It cannot be understated how important this talent is for a medic trying to create high quality genes. This talent shouldn't be nerfed, since it's so important.

- Genetic Genius (Unlock recipe: Gene splicer, reduce the chance to taint when combining genes by 50%)
A really underwhelming final talent for an otherwise great tree. Reducing the taint chance when combining is nice, though genes aren't common enough that medics will ever actually get to use it. It sounds nice on paper, but is practically not often utilized. Even after spending a ridiculous amount of time gathering and refining genes, there's still a 50% chance that your hard work will be turned into tainted genes, which feels devastating. (Well, unless it's singleplayer, in which case everyone save-scums it.)
Also, the advanced splicer can be purchased in stores, thus making this talent even less impactful.
This talent should be better. It's noticeably worse than the other two trees' final talents.
It could be improved by decreasing the combination chance to taint down to 0%. That would give this talent a real strength comparable to the other final talents.

On the topic of genes, there is one major problem in the unstable version. Previously, it was possible to get to 200 medical skill, at which point you could not taint genes when refining. Now, it is not possible to go much over 100. This is a problem, because the tainting chance is rather high, and it makes it extremely difficult to get to high quality genes. There really should be a way to eliminate the RNG factor, especially when it takes such a high amount of work to get genetic materials. It's extremely disheartening when your work is wasted because of a 5% chance to taint.

**Medic**
This tree has some good and some bad talents. A strange design choice to include buffs to assistants in this tree, and another useless CPR talent.

- Medical Expertise (+30 medical skill, better bandages)
This talent is really solid, however the improved bandages won't see much use, since plastiseal is almost always used instead once sufficient resources have been collected. It would be great if the bonus applied to plastiseal as well, even if it was a smaller amount. 
The description could also be a little less vague - something along the lines of "Bandages you apply are 30% more effective" would be greatly useful.

- Emergency Response (Assistants can drag at full speed, you move 40% faster when a crew member is unconscious)
Giving buffs to other characters is fine, but the problem with this talent is that it's 1) only assistants and 2) a talent effect. If there are no assistants on board, then this talent does nothing. If there are assistants, then it invalidates the uniqueness of the medic's fireman's carry ability.

- Medical Assistance (Assistants the same room gain +30 medical skill)
A really bad talent. If the medic is in the same room as an assistant, then the assistant doesn't need more skills, because the medic can do any necessary healing. If the assistant wants to play doctor, then they can pick it as their apprenticeship.
In general, players like to buff _themselves_ with talents, and rarely enjoy spending XP so others can have minor stat boosts.

- Don't Die on Me! (More powerful CPR)
Terrible. A waste of a talent point, honestly. Any medic that could take this talent is guaranteed to be at least 70 medical skill (base 40 + talent 30) so their CPR is going to be absolutely sufficient for any circumstances. CPR is already too efficient to warrant investing a talent point into improving it. This talent should be removed, or CPR should be made more difficult.
I think Dr. Submarine would fit well here, or as replacement for Medical Assistance, if either of those were to be removed.

**Chemist**
An interesting tree. Overall, kinda weak. The one saving grace of this tree is the final talent; endocrine boosters are as fun as ever. But this tree is 90% recipe unlocks, which is a little underwhelming. It could certainly use some secondary effects, since the medic who takes this tree only ends up with a handful of recipes, and not much in the way of buffs or other stat boosts.

Some secondary effects that would synergise/be appropriate for the chemist tree:
- +50% more medical skill gain from fabricating medical items
- +25% burn damage
- When deconstructing crawler eggs or sulphurite shards, output 100% more sulphuric acid

Some secondary effects like these would go a long way in making this tree feel more distinct.

- Delivery System (Unlock recipe: Advanced Syringe Gun)
Not much of an upgrade over the original, this item has 4 more slots, and a health scanner. Neither of these features were really missing on the original. The syringe gun went underutilized because it's just too slow - as a weapon it doesn't really stack up against any of the firearms, and the advanced syringe gun doesn't change that. Even a slight buff to its reload rate would make it more viable, and more competitive with the alternative talent (Lab Contacts - 30% cheaper medicine is really great, and a lot better than the syringe gun - especially since pretty much all medicines will be purchased after plants disappeared from caves.)
Overall could certainly use a buff.

We had some ideas on other talents. Not sure where they might fit in, but perhaps in the Medic tree. Just throwing out ideas, mostly.

- Purity (Reduce medical item side-effects by 50%) This could be changed to only reduce the side effects of a specific type of medicine, like opioids or stimulants.

- Placebo (Whenever you use a medical item, you have a 10% chance to not consume it.) Not entirely certain if this is possible via xml currently.

- [No talent name] (Unlock recipe: Advanced Health HUD) This item would display the health status of crewmembers aboard the submarine.

That's pretty much it. I certainly don't expect all - or much - of this to be acted upon, but I think it's at least worthy of consideration and discussion. Thank you if you read all this.

A massive thank you to Skyline_Robot. This was a collaborative project between the two of us, I just happened to be the one writing this up.

**Comment by ubertpendragon** [comment]
Agree

-XP talents are not good _in general_ and should at least have their values doubled

-La Resistance deserves more than just +10% extra husk

-Don't Die on Me!  is exceptionally useless. It needs to be reworked, CPR needs an ability effect type like OnApplyCPR so we can actually design effects around applying CPR, applying statuseffects to either the medic or the victim _(for example: giving a small amount of Vigor/Hyperactivity...)_

-Advanced Syringe Gun could use an extra 1-syringe slot that takes priority, exactly like Rifle and 40mm grenade, so you have your main 8x syringe mag and then you can quickly insert one specific syringe via hotkey

**Comment by JeroonBosch** [comment]
Thanks for the feedback! We're a bit limited in what we can do for the upcoming patch (all the texts are being translated, no new texts can be added for the upcoming patch), but we'll be looking to tune a bit more where we can.
